contracts/tasks/deploy/poll/01-poll.ts

```diff
@@ -133,7 +135,7 @@ deployment.deployTask("poll:deploy-poll", "Deploy poll").setAction(async (_, hre
     }),
 
     storage.register({
-      id: EContracts.Tally,
+      id: useQuadraticVoting ? EContracts.Tally : EContracts.TallyNonQv,
```

Comment:
Is TallyNonQv deployed with the same constructor args as Tally?

Reply:
yes